### PR TITLE
Upgraded ANF SDK to latest version

### PR DESCRIPTION
**Pull request by AdelGhabboun**
1. Upgraded Azure Netapp files SDK to latest 1.13.0
2. Upgraded .NET Framework to .NET 5.0

### CODE REVIEW

**Review by hildurhodd — APPROVED** [review]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,22 @@
 ## Azure NetAppFiles Cross-Region Replication (CRR) SDK NFS 4.1 Sample .NETCore
 
+[1.0.1 (2021-09-07)](#1.0.1 (2021-09-07))
 [1.0.0 (2019-10-09)](#1.0.0 (2019-10-09))
 
 # 1.0.0 (2019-10-09)
```
Comment (paulomarquesc) [inline CHANGELOG.md:6]: Needs to be updated

**Review by paulomarquesc — CHANGES_REQUESTED** [review]
:clock1:

**Review by paulomarquesc — APPROVED** [review]
:shipit:

**Pull request merged**